src/compiler/crystal/macros.cr

```diff
@@ -646,10 +646,18 @@ module Crystal::Macros
     def unshift : ArrayLiteral
```

Comment:
Shouldn't `#unshift` accept an argument?

Reply:
It does but just isn't documented it looks like.

Reply:
So it should - question is whether in this PR or the new one?

Reply:
fixed in 201c834

Reply:
I'd put into another PR, because one is about adding new features, and another is a fix